middlewares/runtime-errors.js

```diff
@@ -18,11 +18,12 @@ const customOpenApiError = (err, req, res, next) => {
   // call the next middleware function if the error is not an openapi error
   if (!isOpenApiError(err)) {
     next(err);
+  } else {
+    /**
+    * @todo Implement custom OpenAPI error rules and handlers here.
```

Comment:
I think you miss a space before the first `*` at line 23 and 24. In addition, can you add the JIRA ticket number here as well?